pandas/tests/reductions/test_reductions.py

```diff
@@ -960,6 +960,18 @@ def test_min_max(self):
         assert np.isnan(_min)
         assert _max == 1
 
+        cat = Series(Categorical(
+            ["a", "b", "c", "a"], categories=["b", "c", "d"], ordered=True))
```

Comment:
for clarity, can you explicitly use NaNs in the construction? (so it is clear it is about NaNs). 
So something like 

```
       cat = Series(Categorical(
             ["a", "b", np.nan, "a"], categories=['b', 'a'], ordered=True))
```

should be sufficient for the test